Microsoft.Toolkit.Uwp.UI.Controls/ImageEx/ImageEx.Source.cs

```diff
@@ -113,6 +113,8 @@ private static bool IsHttpUri(Uri uri)
             {
                 if (IsCacheEnabled && _isHttpSource)
                 {
+                    var ogUri = _uri;
```

Comment:
I'd suggest that instead of doing it like this, just pass `_uri` as a paramenter of `LoadImageAsync` method.

For all matters and purposes, if the `_uri` changes when you are loading the image, the await'ed part will get cancelled, so there shouldn't be any need to check for the change anyway!